hack/create-operatorhub-prs.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$1
+if [[ $1 == v* ]]; then TAG=$1; else TAG=v$1; fi
+GITHUB_AUTHOR=$2
+COMMUNITY_OPERATORS=community-operators
+COMMUNITY_OPERATORS_PROD=community-operators-prod
+if [[ $3 == false ]]; then DRY_RUN=false; else DRY_RUN=true; fi
+if [ -z "$4" ]; then KOGITO_OPERATOR_DIR=$(pwd); else KOGITO_OPERATOR_DIR=$4; fi
+
+echo "Kogito Operator directory is ${KOGITO_OPERATOR_DIR}"
+
+git remote add upstream https://github.com/kiegroup/kogito-operator.git >/dev/null 2>&1
+git fetch upstream
+git fetch upstream --tags
+echo "Checking out Kogito $TAG"
+git checkout -B kogito-$TAG $TAG
+
+cd /tmp
+
+create_operatorhub_pr() {
+  echo "### Starting changes on $1 repo ####"
+  if [ -d "$1" ];
+  then
+      echo "$1 directory exists."
+      cd $1
+      git fetch --all
+      git checkout main
+      git merge upstream/main
```

Comment:
shouldn't you do just a rebase?
what is the purpose of this merge here?

Reply:
I removed this since now we are using a clean cloned repo every time